api/v1beta2/ibmvpccluster_types.go

```diff
@@ -113,7 +113,11 @@ type VPCNetworkSpec struct {
 	// +optional
 	ResourceGroup *string `json:"resourceGroup,omitempty"`
 
-	// TODO(cjschaef): Complete spec definition (SecurityGroups, VPC)
+	// vpc defines the IBM Cloud VPC.
+	// +optional
```

Comment:
We already have a field named VPC under IBMVPCClusterSpec, so may be lets add a comment whereever it looks appropriate on whats the difference or when to use which field.

Reply:
I added a little more detail to this field and some other fields/structs, as they related to the extended VPC Infra support.